AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -195,6 +195,8 @@ it-actual.ru##.blok[style="text-align: center;"]
 ! newsweek.com#@#.ad_wide
 ||cdn.pelcro.com/js/bab^
 digitaltrends.com#%#AG_abortInlineScript(/googlefc/, 'document.addEventListener');
+@@||hentaistream.com/out/
+hentaistream.com#@#iframe[width="300"][height="250"]
```

Comment:
If everything works fine without exceptions rules then I think that we don't need to add them.
It seems that these rules are enough:
`hentaistream.com##.othercontent`
`hentaistream.com##.skin-hold`

Reply:
It works fine without exceptions. Ok then I delete them.